### PR TITLE
feat: Alert support aria-* in closable

### PR DESCRIPTION
**Pull request by kiner-tang**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [X] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

- https://github.com/ant-design/ant-design/discussions/47188

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Alert support aria-* in closable（已回滚） |
| 🇨🇳 Chinese | 警告提示组件支持在 closable 中配置 aria-*（已回滚） |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [X] Doc is updated/provided or not needed
- [X] Demo is updated/provided or not needed
- [X] TypeScript definition is updated/provided or not needed
- [X] Changelog is provided or not needed

### CODE REVIEW

**Comment by kiner-tang** [comment]
> ## 👁 Visual Regression Report for PR #47446 Failed ❌
> > **🎯 Target branch:** master ([ff6f9c5](https://github.com/ant-design/ant-design/commit/ff6f9c5a8f82a9c510522530ea1bf42b684da40c))
> > 📖 [View Full Report ↗︎](https://antd-visual-diff.oss-cn-shanghai.aliyuncs.com/pr-47446/visualRegressionReport/report.html)
> 
> Image name	Expected	Actual	Diff
> `alert-closable.compact.png`	master: ff6f9c5a8f82a9c510522530ea1bf42b684da40c	current: pr-47446	diff
> `alert-closable.compact.css-var.png`	master: ff6f9c5a8f82a9c510522530ea1bf42b684da40c	current: pr-47446	diff
> `alert-closable.dark.png`	master: ff6f9c5a8f82a9c510522530ea1bf42b684da40c	current: pr-47446	diff
> `alert-closable.dark.css-var.png`	master: ff6f9c5a8f82a9c510522530ea1bf42b684da40c	current: pr-47446	diff
> `alert-closable.default.png`	master: ff6f9c5a8f82a9c510522530ea1bf42b684da40c	current: pr-47446	diff
> `alert-closable.default.css-var.png`	master: ff6f9c5a8f82a9c510522530ea1bf42b684da40c	current: pr-47446	diff
> Check [Full Report](https://antd-visual-diff.oss-cn-shanghai.aliyuncs.com/pr-47446/visualRegressionReport/report.html) for details

变化符合预期

**Review by afc163 — DISMISSED** [review]

File: components/alert/Alert.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ export interface AlertProps {
   /** Type of Alert styles, options:`success`, `info`, `warning`, `error` */
   type?: 'success' | 'info' | 'warning' | 'error';
   /** Whether Alert can be closed */
-  closable?: boolean;
+  closable?: boolean | ({ icon?: React.ReactNode } & React.AriaAttributes);
```
Comment (afc163) [inline components/alert/Alert.tsx:21]:
```suggestion
  closable?: boolean | ({ closeIcon?: React.ReactNode } & React.AriaAttributes);
```

这样是不是好点？https://github.com/ant-design/ant-design/discussions/42828#discussioncomment-6089019

Comment (kiner-tang) [reply components/alert/Alert.tsx:21]: 确实是，我调整一下

**Review by afc163 — APPROVED** [review]

**Comment by kiner-tang** [comment]
@afc163  我突然反映过来，这次的改动，是不是在 feature 分支上比较好呀

**Comment by afc163** [comment]
合都合了，就这样吧。

**Comment by kiner-tang** [comment]
> 合都合了，就这样吧。

好的，那我其他组件的修改也直接在 master 上搞？

**Comment by afc163** [comment]
那别

**Comment by kiner-tang** [comment]
> 那别

好，其他组件我在 feature 上

**Comment by yoyo837** [comment]
撤回加cherry-pick很容易啊，建议合规

**Comment by li-jia-nan** [comment]
> 撤回加cherry-pick很容易啊，建议合规

+1

**Pull request merged**